### ISSUE TITLE
/opt/rh/devtoolset-8/root/usr/libexec/gcc/x86_64-redhat-linux/8/ld: cannot find -ltcl8.6

### ISSUE THREAD

**Issue by RENEK-bool** [The-OpenROAD-Project/OpenROAD-flow-scripts#1448]
### Subject

[Build] for compilation-related issues.

### Describe the bug

system environment：centos7.9

I do the following steps 
git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
cd OpenROAD-flow-scripts
sudo ./setup.sh
./build_openroad.sh --local   ← until this step I get an error

/opt/rh/devtoolset-8/root/usr/libexec/gcc/x86_64-redhat-linux/8/ld: cannot find -ltcl8.6
collect2: error: ld returned 1 exit status

### Expected Behavior

build_openroad.sh completed successfully

### Environment

```shell
Unknown git commit, this is not a git repository.

Please make sure that you have the latest code changes and add the commit
hash in the description.

kernel: Linux 3.10.0-1160.el7.x86_64
os: CentOS Linux 7 (Core)
cmake version 2.8.12.2
CMake Error: The source directory "/tmp/tmp.Y2bvCGQ1LE" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
```

### To Reproduce

Execute the following command in the same system environment

git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
cd OpenROAD-flow-scripts
sudo ./setup.sh
./build_openroad.sh --local

**Comment by RENEK-bool** [comment]
I don’t know if it has something to do with the fact that centos7.9 can only install the latest version of tcl-8.5.13-8.el7.x86_64 through yum, here is only 8.5

**Comment by vijayank88** [comment]
For Centos7 can you source following before running `./build_openroad.sh --local --threads 1`
```
# to enable gcc-8
source /opt/rh/devtoolset-8/enable
# or to enable clang-7
source /opt/rh/llvm-toolset-7.0/enable
```
Still if you're unable complete, recommend to use docker based installation. https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildWithDocker.html

**Comment by RENEK-bool** [comment]
@vijayank88 Thank you for your reply. I installed a new centos7.9 system on VMware Workstation and successfully completed the installation based on your answer. I guess the reason why my previous installation failed is probably because the centos7.9 system I used installed many things, which caused some conflicts. Next, I plan to reinstall the centos7.9 system to complete the installation of OpenROAD.
